scaladoc-testcases/src/tests/snippetTestcase2.scala

```diff
@@ -19,7 +19,7 @@ trait Quotes2[A] {
      val z: zModule = ???
      trait zModule {
       /**
-        * SNIPPET(OUTERLINEOFFSET:25,OUTERCOLUMNOFFSET:10,INNERLINEOFFSET:7,INNERCOLUMNOFFSET:6)
+        * SNIPPET(OUTERLINEOFFSET:25,OUTERCOLUMNOFFSET:10,INNERLINEOFFSET:9,INNERCOLUMNOFFSET:6)
```

Comment:
Those shouldn't be rendered, right?

Reply:
This is information for tests to assert good positions of errors but it's also a part of docstring so I think it'll be rendered.